### PR TITLE
7.2 UAD in Custom Apps | LRDOCS-7024

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-7024, ported the 7.1 article to 7.2. This is updated and refactored to meet our new 7.2 standards. Basically, we'r explaining things separately from the code instructions, so I broke the article into two pieces.

### CODE REVIEW

File: developer/frameworks/articles/user-associated-data/01-managing-uad-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Administrators can [delete or
+anonymize](/docs/7-2/user/-/knowledge_base/u/managing-user-data) User Associated
+Data (UAD), complying with the EU's General Data Protection Regulation (GDPR).
+Out of the box, this tool supports Liferay applications (blogs, web content,
+etc.), but you can also anonymize data stored by your custom applications.
+
+At the core of the anonymization effort, you must identify fields to anonymize
+by attaching anonymization attributes to elements in the `-service` module's
+`service.xml` file. The UAD framework recognizes two kinds of field:
```
Comment (pei-jung) [inline developer/frameworks/articles/user-associated-data/01-managing-uad-intro.markdown]: There's more uad related fields used in the service.xml, but these two mentioned in the docs are the most important ones. You can look for more uad-* fields in https://github.com/liferay/liferay-portal/blob/master/definitions/liferay-service-builder_7_2_0.dtd.

Comment (pei-jung) [reply developer/frameworks/articles/user-associated-data/01-managing-uad-intro.markdown]: Also, this only applies to entities created using the service builder. For custom entities, they can still use the UAD framework by implementing the interfaces directly, namely UADDisplay, UADAnonymizer, and UADExporter

**Comment by rbohl** [comment]
Forwarded: https://github.com/sez11a/liferay-docs/pull/4312

**Comment by rbohl** [comment]
See my review edits here: https://github.com/sez11a/liferay-docs/pull/4312/commits/0e75e9a8a9411cb50ad249894df93037d75ddcf8

**Pull request closed without merging**